.gitignore

```diff
@@ -15,3 +15,5 @@ yarn-debug.log*
 yarn-error.log*
 yarn.lock
 !/yarn.lock
+packages/*/dist
+plugins/*/dist
```

Comment:
Probably better to use the version @michaelsbradleyjr did in https://github.com/embark-framework/embark/pull/1301/files#diff-a084b794bc0759e7a6b77810e01874f2R9

Reply:
Yes totally agree, I suggested same changes in https://github.com/embark-framework/embark/pull/1294